### DISCUSSION TITLE
Data Sources

### DISCUSSION THREAD

**Discussion by AntoniaJost** [CliDyn/climsight#22] [General]
### Biodiversity API
The biodiversity data stems from the Global Biodiversity Information Facility at [https://techdocs.gbif.org/en/openapi/](https://techdocs.gbif.org/en/openapi/).

### Natural Hazard Data
The natural hazard data is based on a selection of natural disasters from the Emergency Events Database (EM-DAT) by the Centre for Research on the Epidemiology of Disasters (CRED). The here used Geocoded Distasters (GDIS) data set is an extension of it, covering the time period from 1960 to 2018. The data is provided as raster data and can be found here, as well as the corresponding documentation: 
Rosvold, E., and H. Buhaug. 2021. Geocoded Disasters (GDIS) Dataset. Palisades, New York: NASA Socioeconomic Data and Applications Center (SEDAC). [https://doi.org/10.7927/zz3b-8y61](https://doi.org/10.7927/zz3b-8y61). Accessed DAY MONTH YEAR. 
Rosvold, E.L., and H. Buhaug. 2021. GDIS, a Global Dataset of Geocoded Disaster Locations. Scientific Data 8:61. [https://doi.org/10.1038/s41597-021-00846-6](https://doi.org/10.1038/s41597-021-00846-6).

### Population Data
The population data is provided by the United Nations, Department of Economic and Social Affairs, Population Division (2022). World Population Prospects 2022, Online Edition and can be accessed at [https://population.un.org/wpp/Download/Standard/CSV/](https://population.un.org/wpp/Download/Standard/CSV/). It is made available under a Creative Commons license CC BY 3.0 IGO.

**Comment by AntoniaJost** [comment]
### Natural Earth Data
The data for the coastlines, land shape, rivers, and lakes that we are using is from Natural Earth Data. It can be accessed at [https://www.naturalearthdata.com/downloads/10m-physical-vectors/](https://www.naturalearthdata.com/downloads/10m-physical-vectors/).  We are using the 10 meter resolution. The datasets that were used in particular are:
* 10m_coastline
* 10m_land
* 10m_rivers_lake_centerlines
* 10m_rivers_australia
* 10m_rivers_europe
* 10m_rivers_north_america
* 10m_lakes
* 10m_lakes_australia
* 10m_lakes_europe
* 10m_lakes_north_america

For all exact links, please have a look at the download_data.sh file.

**Comment by koldunovn** [comment]
Land cover with 10 m resolution: https://worldcover2020.esa.int/ We have to see if one can download the complete dataset and ipscale it to 1 km or so. Or maybe we can use it as is.
